indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -63,17 +80,28 @@ export default function EditableList() {
     setSortedList(_sortList({sortBy, sortDirection}));
   };
 
+  const code = data => (!data ? Translate.string('N/A') : data);
+  const status = data => (_.isEmpty(data) ? 'not_submitted' : data.state);
+
   // eslint-disable-next-line react/prop-types
   const renderCell = ({dataKey, rowIndex}) => {
+    const row = sortedList[rowIndex];
     return (
       <TooltipIfTruncated>
-        <div index={rowIndex} styleName="rowcolumn-tooltip" role="gridcell">
-          {dataKey === 'assigned' && <Icon name="user outline" />}
-          {dataKey === 'status' ? (
-            <StateIndicator index={rowIndex} state={sortedList[rowIndex][dataKey]} circular label />
-          ) : (
-            sortedList[rowIndex][dataKey]
+        <div
+          id={dataKey}
+          index={rowIndex}
+          styleName="rowcolumn-tooltip"
+          role="gridcell"
+          style={dataKey === 'title' ? {display: 'block'} : {}}
+        >
+          {dataKey === 'friendlyId' && row[dataKey]}
+          {dataKey === 'code' && code(row[dataKey])}
+          {dataKey === 'title' && row[dataKey]}
+          {dataKey === 'status' && (
+            <StateIndicator index={rowIndex} state={status(row['editable'])} circular label />
           )}
+          {dataKey === 'editor' && <Icon name="user outline" /> && row['editable'][dataKey]}
```

Comment:
What about a mapping object here?

```js
const renderFuncs = {
  code: renderCode,
  status: renderStatus,
  editor: renderEditor,
};

const data = sortedList[rowIndex][dataKey];
const fn = renderFuncs[dataKey] || (x => x);
```

And then here you can just do `{fn(data)}`